src/app.py

```diff
@@ -2352,72 +2442,98 @@ def _get_all_installed_apps_id():
     return all_apps_ids_formatted
 
 
-def _check_manifest_requirements(manifest: Dict, action: str):
+def _check_manifest_requirements(
+    manifest: Dict, action: str = ""
+) -> Iterator[Tuple[str, bool, object, str]]:
     """Check if required packages are met from the manifest"""
 
-    if manifest["packaging_format"] not in [1, 2]:
-        raise YunohostValidationError("app_packaging_format_not_supported")
-
     app_id = manifest["id"]
-
     logger.debug(m18n.n("app_requirements_checking", app=app_id))
 
-    # Yunohost version requirement
+    # Packaging format
+    yield (
+        "packaging_format",
+        manifest["packaging_format"] in (1, 2),
+        {},
+        "app_packaging_format_not_supported",  # i18n: app_packaging_format_not_supported
+    )
+
```

Comment:
Hum, didn't understand if there was something to do about this?

Reply:
Yeah imho we should keep the initial code with `raise YunohostValidationError`, because if `packaging_format` is for example 3, it's safe to assume that the rest of the code *is* gonna crash because there's no guarantee that for example the `integration` key will still exist (or other keys/code assumptions)